### ISSUE TITLE
Occasionally encounter entity loading error results rolling stock disappear

### ISSUE THREAD

**Issue by lordfriend** [cam72cam/ImmersiveRailroading#549]
```
[11:42:37] [Server thread/ERROR] [FML]: An Entity immersiverailroading:carpassenger(PRR P70 Coach 1) has thrown an exception during loading, its state cannot be restored. Report this to the mod author
 net.minecraft.util.ReportedException: Loading entity NBT
	at net.minecraft.entity.Entity.func_70020_e(Entity.java:1878) ~[vg.class:?]
 	at net.minecraft.entity.EntityList.func_75615_a(EntityList.java:221) ~[vi.class:?]
 	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_186053_a(AnvilChunkLoader.java:601) ~[aye.class:?]
 	at net.minecraft.world.chunk.storage.AnvilChunkLoader.func_186050_a(AnvilChunkLoader.java:528) ~[aye.class:?]
 	at net.minecraft.world.chunk.storage.AnvilChunkLoader.loadEntities(AnvilChunkLoader.java:485) ~[aye.class:?]
 	at net.minecraftforge.common.chunkio.ChunkIOProvider.syncCallback(ChunkIOProvider.java:101) ~[ChunkIOProvider.class:?]
 	at net.minecraftforge.common.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:94) ~[ChunkIOExecutor.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:118) ~[on.class:?]
	at net.minecraft.world.gen.ChunkProviderServer.func_186028_c(ChunkProviderServer.java:89) ~[on.class:?]
 	at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:135) ~[on.class:?]
 	at net.minecraft.world.World.func_72964_e(World.java:310) ~[amu.class:?]
 	at net.minecraft.world.World.func_175726_f(World.java:305) ~[amu.class:?]
 	at net.minecraft.world.World.func_180495_p(World.java:911) ~[amu.class:?]
	at net.malisis.core.util.clientnotif.ClientNotificationManager.notify(ClientNotificationManager.java:88) ~[ClientNotificationManager.class:1.12.2-6.4.0]
 	at net.minecraft.world.World.handler$onNeighborChanged$zza000(World.java:4118) ~[amu.class:?]
 	at net.minecraft.world.World.func_190524_a(World.java) ~[amu.class:?]
 	at net.minecraft.world.World.func_175685_c(World.java:492) ~[amu.class:?]
 	at blusunrize.immersiveengineering.common.blocks.metal.TileEntityFluidPipe.lambda$validate$0(TileEntityFluidPipe.java:185) ~[TileEntityFluidPipe.class:?]
 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_111]
 	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_111]
 	at net.minecraft.util.Util.func_181617_a(SourceFile:46) [h.class:?]
 	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:723) [MinecraftServer.class:?]
 	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
 	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
 	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
 Caused by: java.lang.NullPointerException
 	at cam72cam.immersiverailroading.entity.EntityCoupleableRollingStock.func_70037_a(EntityCoupleableRollingStock.java:137) ~[EntityCoupleableRollingStock.class:?]
 	at net.minecraft.entity.Entity.func_70020_e(Entity.java:1866) ~[vg.class:?]
 	... 25 more
```

This bug isn't easy to reproduce because I still can't figure out what under condition it can happen.
But One thing can be sure is this is related when chunk loading and unloading. maybe some chunloaders included a train partially will result bad consequences?

**Comment by lordfriend** [comment]
It's more likely with the chunkloader

**Comment by cam72cam** [comment]
Fixed in 3aa264f0415de838f402c4b576703250a7c69e31
